packages/bullet/src/BulletItem.tsx

```diff
@@ -137,7 +137,6 @@ export const BulletItem = ({
     const axis = (
         <g transform={`translate(${axisX},${axisY})`}>
             <Axis
-                // @ts-ignore
```

Comment:
Maybe we could defined an `AxisType` and cast?

Reply:
Not sure I understand what you mean here?